client/dashboard/components/collapsible-card/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card, CardBody, __experimentalHStack as HStack, Button } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { chevronDown, chevronUp } from '@wordpress/icons';
+import clsx from 'clsx';
+import { useState } from 'react';
+import './style.scss';
+
+interface CollapsibleCardProps {
+	header?: React.ReactNode;
+	children: React.ReactNode;
+	toggleLabel?: string;
+}
+
+export const CollapsibleCard = ( { header, children, toggleLabel }: CollapsibleCardProps ) => {
+	const [ isCollapsed, setIsCollapsed ] = useState< boolean >( true );
```

Comment:
Should this be configurable? Should we be able to set it on mount?

Reply:
Good point, I added an `initialExpanded` state and other properties to make the component controllable from the outside, in case we want to use it as part of an accordion.